packages/firebase_ml/lib/src/firebase_remote_model.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of firebase_ml;
+
+/// Describes a remote model to be downloaded to the device.
+///
+/// Defines the download conditions of the model,
+/// whether or not to download updated versions of the model,
+/// and the model's name specified by the developer in the cloud console.
+/// https://firebase.google.com/docs/reference/android/com/google/firebase/ml/common/modeldownload/FirebaseRemoteModel
+class FirebaseRemoteModel {
+  String _modelName;
```

Comment:
can be final

Reply:
It cannot be final with the existing builder configuration because we are initializing it through FirebaseCustomRemoteModel builder i.e. it is not always initialized if the class is called on its own.

Reply:
After discussion with Elvis removed builders in favour of Dart syntactic sugar. This class is now abstract and modelName is public and final.